01_login.php

```diff
@@ -73,29 +95,46 @@
 		<!-- Navbar (sit on top) -->
 		<div class="w3-top">
 			<div class="w3-bar w3-white w3-padding w3-card">
+				<div class="w3-left">
 				<img src="images/placeholder.jpg" width="40" height="40">
+				</div>
 				<a class="w3-bar-item"><b>Soundwave</b> </a>
 			</div>
 		</div>
+		
 
 
 		
 		<!-- background container; colour orange -->
-		<div class="container2" style="max-width:100%">
-		
+		<div class="w3-orange container w3-image">
+			<div class="bg">
+
+
+				<br>
+				<br>
+				<br>
+				<br>
+
+
+				<center>
+					<div class= "container w3-panel w3-padding-large-16" style="height:80%;width:80%; background-color:rgba(255, 255, 255, .2);">
+
+					
 
 			<!-- Container (placeholder) -->
 			<div class="w3-content w3-container w3-padding-64">
 
 				<div class="w3-row">
 					<div class="w3-col m6 w3-center w3-padding-large">
-						<img src="images/placeholder.jpg" class="w3-round w3-image w3-opacity" alt="Photo of Me" width="500" height="333">
+						<img src="images/placeholder.jpg" class="w3-round w3-image" alt="logo brand" width="350" height="233">
```

Comment:
removed opacity effect as it was not needed and edited the alt tag to logo brand. 
Changed the width and height to my liking; this made the placeholder for the logo inlined with the other information